docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+.. tip::
+
+    The use of the color bands allows you to later select the *Green* and the *Swir1* band as TMASK bands for CCDC's internal, multi-temporal cloud removal. You find this in the :guilabel:`OPT` button pop-up menu under :guilabel:`MORE`.
+
+If the creation of the CCDC asset is aimed at the detection of both, forest degradation and deforestation, the *Normalized Degradation Forest Index* NDFI might be another suitable choice as applied by `Bullock et al 2020 <https://www.sciencedirect.com/science/article/pii/S0034425718305200>`_. Note that this article and the NDFI are specifically tested over tropical rainforest of the Brazilian Amazon. Changes in other forest types might be better captured by different ratios or the color bands. For instance one can consider the *Normalized Differenced Moisture Index* NDMI when looking at Mangrove forests.
+
+.. tip::
+    In case of doubt, we suggest to go for the default option and use all of the color bands, except the blue one.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_color_breakbands.png
+    :title: Sensor Selection - Color breakpoint bands
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_ndfi_breakband.png
+    :title: Sensor Selection - NDFI breakpoint band
+    :width: 49%
+    :group: ccdc-asset-recipe
+
+Radar Data
+""""""""""
+
+In order to create a CCDC asset based on underlying radar time-series, you need to select the :guilabel:`RADAR` button. This will make use the Sentinel-1 C-Band SAR Image Collection in Google Earth Engine. To the best of our knowledge, no scientific studies have been done that investigate the ideal band selection for breakpoint detection. As a starting point we suggest to use the default option that includes the VV and the VH band.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_radar.png
+    :title: Sensor Selection - Radar
+    :width: 49%
+    :align: center
+    :group: ccdc-asset-recipe
+
+Planet data
+"""""""""""
+
+For the creation of a CCDC asset based on Planet data, you have the choice of either selecting the Planet custom basemaps (including the NICFI Level 1 data), or daily imagery itself.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/sensor_selection_planet.png
+    :title: Sensor Selection - Planet
+    :width: 49%
+    :align: center
+    :group: ccdc-asset-recipe
+
+|
+In both cases, the data already needs to reside within Earth Engine as an *ImageCollection asset*, whose ID needs to be filled in the respective field.
+
+In case you want to use the NICFI Level-1 basemaps, you can use the already existing assets within Earth Engine, given that you enabled the access feature as explained `here <https://developers.planet.com/docs/integrations/gee/nicfi/>`_. The NICFI Level-1 assets are splitted by continent and have the following Asset IDs:
```

Comment:
linked to #102 so link to our nicfi page instead, I'll update the doc soon (the presentation is already ready)